Source/AlertController.swift

```diff
@@ -108,7 +108,7 @@ public class AlertController: UIViewController {
     public var shouldDismissHandler: (AlertAction? -> Bool)?
 
     /// The visual style that applies to the alert or action sheet.
-    public lazy var visualStyle: DefaultVisualStyle = DefaultVisualStyle(alertStyle: self.preferredStyle)
+    public lazy var visualStyle: AlertVisualStyle = AlertVisualStyle(alertStyle: self.preferredStyle)
```

Comment:
I thought so too, but it is. Removing it leads to `Value of type 'NSObject -> () -> AlertController' has no member 'preferredStyle'`